production.yml

```diff
@@ -36,18 +36,8 @@ services:
     networks:
       - protonmail
       - default
-
-  traefik:
-    build:
-      context: .
-      dockerfile: ./compose/production/traefik/Dockerfile
-    image: pra_request_tracker_production_traefik
-    depends_on:
-      - django
-    volumes:
-      - production_traefik:/etc/traefik/acme:z
     ports:
-      - "6667:80"
+      - "6667:5000"
```

Comment:
This diff is a ltitle hard to read but I moved the `ports` directive into the django service.